Sample/UIKit/SecondPageViewController.xib

```diff
@@ -20,14 +20,14 @@
             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
             <subviews>
                 <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="roI-sB-TSB">
-                    <rect key="frame" x="137" y="800" width="140" height="30"/>
+                    <rect key="frame" x="137" y="834" width="140" height="30"/>
                     <state key="normal" title="Return to homepage"/>
                     <connections>
                         <action selector="returnToHomepage:" destination="-1" eventType="touchUpInside" id="amB-Mg-c5j"/>
                     </connections>
                 </button>
                 <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Second page" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ys1-Pr-pdf">
-                    <rect key="frame" x="156.5" y="76" width="101" height="21"/>
+                    <rect key="frame" x="156.66666666666666" y="32" width="100.99999999999997" height="21"/>
```

Comment:
My perfectionism! 😱 😄

Reply:
Unintentionally added it because I open the Xib file 😅
Change back here [b5f252c](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/b5f252c174d847bec26bc3c47357b26ee92e7790).